openy_activity_finder.info.yml

```diff
@@ -4,3 +4,6 @@ description: Framework for Activity Finder functionality.
 package: OpenY
 version: 4.0.0
 core_version_requirement: ^8 || ^9
+dependencies:
+  - drupal:rest
+  - serialization:serialization
```

Comment:
I believe `drupal:rest` should be enough here - it requires serialization.
If not - then the correct string for serialization module should be `drupal:serialization` as it is included into drupal core.

Reply:
Nice catch, agree